firetv/__init__.py

```diff
@@ -116,57 +116,215 @@ def connect(self):
                 signer = Signer(self.adbkey)
 
                 # Connect to the device
-                self._adb = adb_commands.AdbCommands().ConnectDevice(serial=self.host, rsa_keys=[signer])
+                self.adb = adb_commands.AdbCommands().ConnectDevice(serial=self.host, rsa_keys=[signer])
             else:
-                self._adb = adb_commands.AdbCommands().ConnectDevice(serial=self.host)
+                self.adb = adb_commands.AdbCommands().ConnectDevice(serial=self.host)
         except socket_error as serr:
             logging.warning("Couldn't connect to host: %s, error: %s", self.host, serr.strerror)
 
+    def app_state(self, app):
+        """ Informs if application is running """
+        if not self.adb or not self.screen_on:
+            return STATE_OFF
+        if self.current_app["package"] == app:
+            return STATE_ON
+        return STATE_OFF
+
+    def launch_app(self, app):
+        if not self.adb:
+            return None
+
+        return self._send_intent(app, INTENT_LAUNCH)
+
+    def stop_app(self, app):
+        if not self.adb:
+            return None
+
+        return self._send_intent(PACKAGE_LAUNCHER, INTENT_HOME)
+
+    # ======================================================================= #
```

Comment:
The presence of these breaks suggest to me that we should break up this file.

Reply:
I like it the way it is.  These breaks simply help a user to figure out what methods are available and whether they're low level (like the ADB methods) or high level (like the "key" methods).